package.json

```diff
@@ -18,13 +19,19 @@
     "build": {
       "dependencies": [
         "submodules",
-        "./packages/realm:build",
+        "bundle",
+        "./packages/realm:build"
+      ]
+    },
+    "bundle": {
+      "dependencies": [
+        "./packages/realm:bundle",
         "./packages/babel-plugin:build",
         "./packages/mocha-reporter:build",
+        "./packages/realm-app-importer:build",
```

Comment:
At some point we should align on the name of this script, as it is `bundle` in `realm` and `build` everywhere else.

Reply:
Yeah. I agree, its a little strange to have different names there. It's just that the convention I've seen elsewhere is "build", but we want to separate this in our packages.
We should probably rename this from "build" to "bundle" in the packages that doesn't have native code 👍

Reply:
I think generally in pure JS produces, build means bundle.  I guess for our case, the term `bundle` is more transparant.